modules/core/src/main/java/org/apache/ignite/internal/jdbc/thin/JdbcThinConnection.java

```diff
@@ -1403,7 +1413,7 @@ private JdbcThinTcpIo cliIo(List<UUID> nodeIds) {
             return txIo;
 
         if (nodeIds == null || nodeIds.isEmpty())
-            return iosArr[RND.nextInt(iosArr.length)];
+            return randomIo();
 
         JdbcThinTcpIo io = null;
```

Comment:
Is there a need to call `nodeIds.iterator().next()` couple of lines below? Can we call `nodeIds.get(0)` instead?

Reply:
Yep, I believe It's an atavism of `Collection<UUID>` that was later reworked to `List<UUID>`.